R/check_dimensions.R

```diff
@@ -216,6 +216,10 @@ tblcheck_message.length_problem <- function(problem, ...) {
     }
   }
 
+  if (!length(problem$value)) {
+    return(glue::glue_data(problem, problem$exp_msg, problem$obj_msg))
+  }
+
   glue::glue_data(problem, problem$exp_msg, problem$obj_msg, problem$value_msg)
```

Comment:
What if this were...?

```suggestion
  glue::glue_data(
    problem, 
    problem$exp_msg, 
    problem$obj_msg, 
    if (!length(problem$value)) problem$value_msg else ""
  )
```

or also this version?

```suggestion
  if (!length(problem$value)) {
    # remove values message: obj and exp have the same values but different length
    problem$value_msg <- ""
  }

  glue::glue_data(problem, problem$exp_msg, problem$obj_msg, problem$value_msg)
```

I like that these versions bring the focus to the `value_msg` and highlight that `exp_msg` and `obj_msg` don't change.